rs/src/main.rs

```diff
@@ -0,0 +1,120 @@
+use alloy::{
+    eips::eip7702::Authorization,
+    network::{TransactionBuilder, TransactionBuilder7702},
+    primitives::{Address, Bytes, TxKind, U256, address},
+    providers::{Provider, ProviderBuilder},
+    rpc::types::TransactionRequest,
+    signers::{SignerSync, local::PrivateKeySigner},
+    sol,
+};
+use bytes::{BufMut, BytesMut};
+use eyre::Result;
+use std::env;
+use std::str::FromStr;
+
+sol!(
+    #[allow(missing_docs)]
+    #[sol(rpc)]
+    EOAMultisend,
+    "../out/EOAMultisend.sol/EOAMultisend.json"
+);
+
+async fn eoa_multisend(pk: PrivateKeySigner, calls: Vec<TransactionRequest>) -> Result<()> {
+    let rpc_url = "https://sepolia.drpc.org".parse()?;
+    let provider = ProviderBuilder::new()
+        .wallet(pk.clone())
+        .connect_http(rpc_url);
+
+    let contract = EOAMultisend::deploy(&provider).await?;
```

Comment:
Can we assume its already deployed and instantiate an instance from address? Soon we will create a script to deploy this same contract to every network with the same address (similar to the existing multisend).

Reply:
We probably can. Let me look into it. Should I use the address at `EOA_MULTISEND_ADDRESS` in the ts script?

Reply:
Done! 👍🏽